modules/caddytls/internalissuer.go

```diff
@@ -114,12 +119,16 @@ func (iss InternalIssuer) Issue(ctx context.Context, csr *x509.CertificateReques
 
 	// ensure issued certificate does not expire later than its issuer
 	lifetime := time.Duration(iss.Lifetime)
-	if time.Now().Add(lifetime).After(issuerCert.NotAfter) {
-		// TODO: log this
-		lifetime = issuerCert.NotAfter.Sub(time.Now())
+	if time.Now().Add(time.Duration(iss.Lifetime)).After(issuerCert.NotAfter) {
```

Comment:
Shouldn't this use `lifetime`?

Reply:
You're right - this was left over from some experimentation I was doing 😅